WEB_SERVICE/WEB/_MEDIUM/INSECURE_FORWARD_SECRECY/recommendation.md

```diff
@@ -0,0 +1,13 @@
+To mitigate the risk of decrypting past communications due to the absence of Forward Secrecy, consider the following recommendations:
+
+**Enable Forward Secrecy:** Implement key exchange algorithms that support Forward Secrecy, such as ECDHE or DHE. These algorithms ensure that unique session keys are generated for each connection, making it impossible to decrypt past communications even if long-term keys are compromised.
+
+**Use Strong Encryption Suites:** Prefer cipher suites that combine ECDHE or DHE with strong encryption methods like AES-GCM or ChaCha20-Poly1305. By prioritizing these strong suites, you enhance the overall security of your encrypted communications.
+
+**Use TLS 1.2 or TLS 1.3:** Always use TLS 1.2 or TLS 1.3, as TLS 1.3 enforces Forward Secrecy by default, simplifying the configuration and reducing the risk of misconfiguration. For TLS 1.2, ensure that only cipher suites supporting Forward Secrecy are enabled.
+
+**Disable Non-FS Cipher Suites:** Regularly audit your server configuration to remove any cipher suites that do not support Forward Secrecy, such as those utilizing RSA key exchange or other static key methods. This will help minimize the risk of potential vulnerabilities.
+
+**Regularly Review TLS Configuration:** Continuously monitor and update your TLS configuration to comply with the latest security recommendations and best practices. This proactive approach will ensure that your server remains secure against evolving threats.
+
```

Comment:
Also it would be better to add some configuration snippet for common web servers (like nginx, apache, ...)